### PR TITLE
added contributors page

### PR DESCRIPTION
**Pull request by aysal04**
## Description

Creatied a seperate page containing the images and info about contributors of of this project. It addresses issue #222 

## Pull Request Checklist

Please make sure that your PR meets the following requirements:

- [x] The app is working correctly after applying the changes.
- [x] The app is responsive and displays properly on different screen sizes and devices.
- [x] The changes are thoroughly documented in the appropriate files or documentation sections.
- [x] The coding conventions and style guidelines are followed consistently.

## Checklist

Please check all applicable boxes and provide any additional information if needed.

- [x] I have tested the changes locally and ensured that the app is functioning as expected.
- [x] I have verified that the app is responsive and looks good on various screen sizes and devices.
- [x] I have reviewed the documentation and ensured that it accurately reflects the changes made.
- [x] I have followed the coding conventions and style guidelines defined in the project.
- [x] I have run the automated tests and all of them pass.
- [x] I have considered the performance implications of the changes made.
- [x] I have considered the security implications of the changes made.
- [x] I have checked for any potential regressions or side effects caused by the changes.
- [x] I have performed a self-review of my code.
- [x] Starred the repository.
- [x] Added My Name to the Contributor's List.
- [x] Followed the repository's [Contributing Guidelines](/CONTRIBUTING.md).
- [x] I ran the app and tested it locally to verify that it works as expected.

# Screenshots - If Any (Optional)

image

## Reviewers

@tusharbansal22

### CODE REVIEW

**Comment by AmanJain18** [comment]
I think its available for ur local deployment, we can't see contributors data.
Hint - Check Request Properly

**Comment by AmanJain18** [comment]
Merge this @anmode, i will fixed the issue later, as he will not have acces to it.. Assign Level 3 to it

**Comment by tusharbansal22** [comment]
I guess the issue is that the apikey is in .env file that is why it wont be loading on our devices

File: pages/contributors/index.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import Footer from "components/common/Footer";
+import Header from "components/common/Header/Header";
+import styles from "@styles/scss/contributor.module.scss";
+const Contributors = () => {
+  const [contributors, setContributors] = useState([]);
+  useEffect(() => {
+    const fetchContributors = async () => {
+      try {
+        const response = await fetch(
+          "https://api.github.com/repos/aysal04/Project-Amupedia/contributors",
```
Comment (tusharbansal22) [inline pages/contributors/index.js:13]: change this link to the actual repo link, not your fork

**Review by tusharbansal22 — APPROVED** [review]
Great work, one of the actually good PRs out there, just change the link

File: pages/contributors/index.js (hunk 1 above)
Comment (AmanJain18) [reply pages/contributors/index.js:13]: I will fix it...

**Review by AmanJain18 — APPROVED** [review]

**Comment by tusharbansal22** [comment]
Congrats @aysal04 , looking forward to more contribution from your end.

**Pull request merged**